views/standard/spec/columns.js

```diff
@@ -179,6 +203,18 @@ define(['jquery', 'appearanceUtils'], function($, utils) {
 
             var signatures = pull.qa_signatures;
 
+            if (pull.milestone) {
+               var due_date = pull.milestone.due_on &&
+                new Date(pull.milestone.due_on);
+
+               // If milestone is past due date.
+               if (due_date && due_date < new Date()) {
+                  score -= 2000;
+               } else {
+                  score -= 1000;
+               }
+            }
+
```

Comment:
Aha! As long as this function does what I think it does, I should be able to add the logic to one place.

https://github.com/iFixit/pulldasher/blob/master/views/standard/spec/index.js#L33